recipes/example/meta.yaml

```diff
@@ -9,12 +10,13 @@
 #  `conda install openssl -c conda-forge``
 
 package:
-  name: simplejson
+  name: {{ name|lower }}
```

Comment:
do you need to add a `name.replace("-", "_")` here, or whatever the equivalent is in jinja syntax?